pylab/website/urls.py

```diff
@@ -11,6 +11,8 @@
     url(r'^projects/create/$', views.project_create, name='create-project'),
     url(r'^projects/(?P<project_slug>%s)/$' % slug, views.project_details, name='project-details'),
     url(r'^projects/(?P<project_slug>%s)/update/$' % slug, views.project_update, name='project-update'),
+    url(r'^events/(?P<year>[0-9]{4})/(?P<month>[0-9]{2})/(?P<day>[0-9]+)/(?P<event_slug>%s)/$' % slug,
+        views.event_details, name='event-details'),
```

Comment:
Well since we decided to hardcode this for now, so it is not only naming but also the date in view is hardcoded to Monday. In this case I think it is OK to explicitly name it with `monday`.